packages/react/src/types/component-props.ts

```diff
@@ -533,12 +653,12 @@ export interface ReactFlowProps<NodeType extends Node = Node, EdgeType extends E
    */
   autoPanSpeed?: number;
   /**
-   * You can enable this prop to automatically pan the viewport while making a new connection.
-   * @default true
+   * The radius around a handle where you drop a connection line to create a new edge.
+   * @default 20
```

Comment:
totally wrong description and default value

https://github.com/xyflow/xyflow/blob/b9e4f829339c2e3dd489dba2f207e11fb7c67f7c/packages/react/src/store/initialState.ts#L123